docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -383,52 +402,7 @@ When an E2E test case fails, it can be helpful to see the screenshot of the appl
 
 #### Take screenshots for failed tests
 
-[Detox supports taking in-test screenshots of the device](https://wix.github.io/Detox/docs/api/screenshots). It exposes the `device.takeScreenshot()` function that can be called from any test case.
-
-Neither `jest` nor `detox` offers a simple way to detect when a particular test case has failed and take a screenshot for only failed test cases. You will have to implement your own mechanism for that.
-
-Edit **e2e/environment.js** and add the `handleTestEvent` method to the `CustomDetoxEnvironment` class. The function handles test events and sets a global `testFailed` variable for test case failure. See the snippet below:
-
-```ts e2e/environment.js
-class CustomDetoxEnvironment extends DetoxCircusEnvironment {
-  // ...
-
-  async handleTestEvent(event, state) {
-    const { name } = event;
-
-    if (['test_start', 'test_fn_start'].includes(name)) {
-      this.global.testFailed = false;
-    }
-
-    if (name === 'test_fn_failure') {
-      this.global.testFailed = true;
-    }
-
-    await super.handleTestEvent(event, state);
-  }
-}
-```
-
-After modifying the environment class, define an `afterEach` hook that calls `device.takeScreenshot()` for failed tests. Create the **e2e/setup.js** file with the following snippet:
-
-```ts e2e/setup.js
-afterEach(async () => {
-  if (testFailed) {
-    await device.takeScreenshot('screenshot');
-  }
-});
-```
-
-The last step is to configure `jest` to load `e2e/setup.js` before running tests. This way, you don't need to include the `afterEach` hook in every test suite:
-
-```json e2e/config.json
-{
-  /// ...
-  "setupFilesAfterEnv": ["./setup.js"]
-}
-```
-
-After making those changes, screenshots for failed tests will be saved in the `artifacts` directory.
+Detox supports taking in-test screenshots of the device. The **detox.config.js** sample above includes a line to configure Detox to take screenshots of failed tests.
```

Comment:
```suggestion
Detox supports taking in-test screenshots of the device. The [**detox.config.js** sample](/build-reference/e2e-tests/#configure-detox) above includes a line to configure Detox to take screenshots of failed tests.
```

Let's add a link to the section that includes this code snippet so that folks as a reference.